### DISCUSSION TITLE
posts/how-to-create-a-react-app-with-rust-and-wasm/?giscus=feea067740fc1bf07861fc0fWEM47ccYjVxBrGVFZPd5d4tT3mYxZxVWE0LRi5HuelITd%2BDDl4xahsuOjR2AjMUU5GtsdXqahc3qAGmv8GEjT0Dxer6tgEyBwh0PkKS6Yifsu4sHGWUPkLrhERk%3D

### DISCUSSION THREAD

**Discussion by giscus** [tkat0/tkat0.github.io#9] [Blog Comments]
# posts/how-to-create-a-react-app-with-rust-and-wasm/?giscus=feea067740fc1bf07861fc0fWEM47ccYjVxBrGVFZPd5d4tT3mYxZxVWE0LRi5HuelITd%2BDDl4xahsuOjR2AjMUU5GtsdXqahc3qAGmv8GEjT0Dxer6tgEyBwh0PkKS6Yifsu4sHGWUPkLrhERk%3D

In this article, I'll introduce followings through creating simple demo application.

https://www.tkat0.dev/posts/how-to-create-a-react-app-with-rust-and-wasm/

**Comment by rnofenko** [comment]
Thanks, great instructions.

**Comment by AN1001** [comment]
Nice, easy to understand and useful!
